libraries/protocol/include/golos/protocol/steem_operations.hpp

```diff
@@ -515,8 +515,8 @@ namespace golos { namespace protocol {
             /**
              * Fee for breaking referral deductions by referral
              */
-            asset referral_break_fee =
-                    asset(GOLOS_DEFAULT_REFERRAL_BREAK_FEE, STEEM_SYMBOL);
+            asset max_referral_break_fee =
+                    asset(GOLOS_DEFAULT_MAX_REFERRAL_BREAK_FEE, STEEM_SYMBOL);
```

Comment:
I described GOLOS_DEFAULT_MAX_REFERRAL_BREAK_FEE as asset, so there should be GOLOS_DEFAULT_MAX_REFERRAL_BREAK_FEE without asset